app/javascript/packs/employee.js

```diff
@@ -0,0 +1,77 @@
+// Arrays of strings for the date display
+const days = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+const months = [
+	'Jan', 'Feb', 'Mar', 'Apr', 
+	'May', 'Jun', 'Jul', 'Aug', 
+	'Sep', 'Oct', 'Nov', 'Dec'
+];
+
+// Get the DOM elements
+const dateDisplay = document.getElementById('date');
+const hourHand = document.getElementById('hour-hand');
+const minuteHand = document.getElementById('minute-hand');
+const secondHand = document.getElementById('second-hand');
+const digitalTime = document.getElementById('digital-time');
+
+let darkMode = localStorage.getItem('darkmode');
+// console.log(darkMode);
+if (darkMode == 'yes') {
+	document.body.classList.add('dark');
+	document.getElementById('dark-mode-toggle').checked = 'true';
+}
+
+// Initialise second hand
+let timeNow = new Date();
+let seconds = timeNow.getSeconds();
+let sRot = seconds * 6 - 96;
+
+// Get and display the time
+const clock = function() {
+	
+	let timeNow = new Date();
+	// DEBUG: new Date(year, month, day, hours, minutes, seconds)
+	// timeNow = new Date(2019,7,21,18,22,41);
+	// DEBUG end
+	
+	// Display the date
+	let day = timeNow.getDay();
+	let date = timeNow.getDate();
+	let month = timeNow.getMonth();
+	dateDisplay.innerHTML = `${days[day]}, ${format(date)} ${months[month]}`;
+	
+	// Each second is 6 degrees of arc.
+	// Second hand moves in discrete steps
+	let seconds = timeNow.getSeconds();
+	// let sRot = seconds * 6 - 90;
```

Comment:
If you are not using code, it is best to remove it before uploading it to GitHub.

Reply:
Deleted code.